From 3fbb93e0ce5f78cd7bb864b0dd2d686071b2759e Mon Sep 17 00:00:00 2001
From: Arjan van de Ven <arjan@linux.intel.com>
Date: Thu, 4 Aug 2016 18:07:50 +0000
Subject: [PATCH 52/55] free initmem asynchronously

Signed-off-by: Sasha Levin <alexander.levin@microsoft.com>
---
 init/main.c | 9 +++++++--
 1 file changed, 7 insertions(+), 2 deletions(-)

diff --git a/init/main.c b/init/main.c
index a8100b954839..6f7006acca3c 100644
--- a/init/main.c
+++ b/init/main.c
@@ -992,6 +992,12 @@ static inline void mark_readonly(void)
 }
 #endif
 
+static void async_free_initmem(void *data, async_cookie_t cookie)
+{
+	free_initmem();
+	mark_readonly();
+}
+
 static int __ref kernel_init(void *unused)
 {
 	int ret;
@@ -1000,8 +1006,7 @@ static int __ref kernel_init(void *unused)
 	/* need to finish all async __init code before freeing the memory */
 	async_synchronize_full();
 	ftrace_free_init_mem();
-	free_initmem();
-	mark_readonly();
+	async_schedule(async_free_initmem, NULL);
 	system_state = SYSTEM_RUNNING;
 	numa_default_policy();
 
-- 
2.14.1

